contracts/auction.sol

```diff
@@ -260,6 +262,7 @@ contract DutchAuction {
         isValidPayload
         atStage(Stages.AuctionEnded)
     {
+        require(now > end_time + waiting_period);
```

Comment:
Can you add a comment here explaining the logic behind this wait? To give us time to react if something happens.